src/views/accounts/AccountActivity/AccountActivityList.vue

```diff
@@ -71,7 +71,27 @@ export default {
         hasImport: false,
         hasDatePicker: true,
         searchConfig: {
-          exclude: ['resource_type']
+          exclude: ['resource_type', 'action'],
+          options: [{
+            value: 'action',
+            label: this.$t('Action'),
+            type: 'choice',
+            children: [
+              {
+                default: true,
+                value: 'view',
+                label: this.$t('View')
+              },
+              {
+                value: 'update',
+                label: this.$t('Update')
+              },
+              {
+                value: 'delete',
+                label: this.$t('Delete')
+              }
+            ]
+          }]
         }
       }
     }
```

Comment:
I'm sorry, but your message is quite unclear. Could you please clarify what the code needs checking? Do you want to analyze differences between current and past versions of JavaScript or suggest any improvements for a project that I have not provided here?